Docker/ods-api-web-sandbox-admin/alpine/pgsql/appsettings.template.json

```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
     "EdFi_Ods": "host=${ODS_POSTGRES_HOST};port=${POSTGRES_PORT};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_{0};pooling=${NPG_POOLING_ENABLED};maximum pool size=${NPG_SANDBOX_MAX_POOL_SIZE_ODS};application name=EdFi.Ods.SandboxAdmin",
-    "EdFi_Security": "host=${ADMIN_POSTGRES_HOST};port=${POSTGRES_PORT};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Security;pooling=${NPG_POOLING_ENABLED};maximum pool size=${NPG_SANDBOX_MAX_POOL_SIZE_SECURITY};application name=EdFi.Ods.SandboxAdmin",
+    "EdFi_Security": "host=${ADMIN_POSTGRES_HOST};port=${POSTGRES_PORT};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=EdFi_Admin;pooling=${NPG_POOLING_ENABLED};maximum pool size=${NPG_SANDBOX_MAX_POOL_SIZE_SECURITY};application name=EdFi.Ods.SandboxAdmin",
```

Comment:
What happens if the connection strings are all you change? Will the deploy process for ODS/API 7.x work with the same connection string in both `EdFi_Admin` and `EdFi_Security`?